helm/syntho-ui/values.yaml

```diff
@@ -100,6 +100,7 @@ frontend:
         - hosts:
           - syntho.company.com
           secretName: frontend-tls
+      ingressAnnotation: ""
```

Comment:
So we will create a seperate variable to keep this particular annotation in? Should we then not rename it TLS or SSL annotation?

Reply:
- So we will create a seperate variable to keep this particular annotation in? 
I'd like to hear your opinion on this.

- Should we then not rename it TLS or SSL annotation?
it is coming from `.Values.frontend.ingress.tls.ingressAnnotation`, AFAIU from your comment, it just could be `.Values.frontend.ingress.tls.annotation` cuz its parent is `tls` already.

Reply:
`.Values.frontend.ingress.tls.annotation` is indeed fine instead of `ingressAnnotation`. 

I think generally this should be fine, but as it's possible to add multiple annotations we'd want to have that working in the future probably as well. The Helm chart still accepts the other annotations as well, so I think this should be fine. In the future we might want to add the capability of adjusting the existing annotations or adding additional annotations not related to the ssl certificate.